inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/PdfExtractFile.java

```diff
@@ -42,22 +45,94 @@
     private String stringContent;
 
     /**
-     * Contains position mapping for a character between PDFExtract string
+     * Contains position mapping for characters between PDFExtract string
      * including and excluding Draw Operations.
      */
     private BidiMap<Integer, Integer> stringPositionMap;
 
+    private Map<String, String> ligatures;
```

Comment:
What is the purpose of this map?

Reply:
It contains the mappings that were extracted from `substitutionTable.xml`